sisl/viz/GUI/README.md

```diff
@@ -0,0 +1,26 @@
+This is the graphical interface.
```

Comment:
This readme file should be moved to the documentation `docs/visualization`, add an entry there.

Reply:
I'm trying to add things to docs, but I have never used this rst file format. Is there a way I can quickly check how the docs look?

Reply:
Check the other rst documents, I'll work them through (when I get the time) :)